### DISCUSSION TITLE
60ms audio latency

### DISCUSSION THREAD

**Discussion by okolenmi** [ppy/osu#22917] [Q&A] [answered]
I have a trouble both in osu!stable and lazer. Hitsounds activation is very slow so I can't play comfortably 6*+ stars maps with hitsounds enabled. 
I have recorded audio with hit and then hitsound and trying to figure out what's wrong.There is an example of how slow system works for hits:
![изображение](https://user-images.githubusercontent.com/55319814/226252735-78d80cd9-adc9-45f7-b963-588e3989076d.png)

Archive with original hitsound and part of recorded audio:
[hit.zip](https://github.com/ppy/osu/files/11014418/hit.zip)

**As you can see, I have 60ms audio latency.**

Latency of my keyboard at the moment is 1-10ms. I have a good feeling of game without sound enabled - so it's not caused by input devices delay. It looks like problems with output.

What's wrong with this? I have nothing except default audio drivers, system is Windows 10.

**Comment by peppy** [comment, marked as answer]
This is highly depending on update frame rate and audio hardware. 60ms sounds a bit higher than average, but also within acceptable range (consider you are including keyboard, usb polling, game update, audio output in your testing).

We have some upcoming changes which will reduce this across all systems, so keep a look out for that.

**Reply by okolenmi** [reply]
Well, I hope that chages will help reduce latency to an acceptable level to play with hitsouns on my system. I have no other latency problems except this one. I'm wondering this problem can be caused by sound library that was used in game or not. Thank you for answer.

**Reply by Axilotl17** [reply]
> I have a trouble both in osu!stable and lazer

then likely not an osu! problem. hopefully ppy will work his magic and make it better anyways, however.
